File: nodejs/formattingText.js
class Formatter{
    constructor(textMessage){
        this.text = String(textMessage);
    }

    bold(){
        return `*${this.text}*`;
    }

    italic(){
        return `_${this.text}_`;
    }

    add_link(link){
        return `[${this.text}](${link})`;
    }

    add_details(details){
        return "```["+this.text+"]"+details+"```";
    }
}

module.exports = {
    Formatter
}

File: python/formattingText.py
class formatObject(object):
    def __init__(self, text_message: str):
        self.tm = text_message

    @property
    def bold(self):
        return f"*{self.tm}*"
    
    @property
    def italic(self):
        return f"_{self.tm}_"
    
    def add_link(self, link: str):
        return f"[{self.tm}]({link})"
    
    def add_info(self, details: str):
        return f"```[{self.tm}]{details}```"